src/components/shared/editable-detail/editable-detail.test.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { render, screen } from '../../../test-support/test-utils';
+import EditableDetail from './editable-detail';
+import { EditModeContext } from '../../projects/wbs-details/work-package-container/work-package-container';
```

Comment:
This line had errors and wouldn't compile when I pulled it down to my local env. You'll want to examine the context hooks pattern